src/o365/spo/commands/page/page-remove.ts

```diff
@@ -0,0 +1,201 @@
+		const telemetryProps: any = super.getTelemetryProperties(args);
+		return telemetryProps;
+	}
+
+	public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+		const resource: string = Auth.getResourceFromUrl(args.options.webUrl);
+		let siteAccessToken: string = '';
+		let requestDigest: string = '';
+		let pageName: string = args.options.name;
+		const serverRelativeSiteUrl: string = args.options.webUrl.substr(args.options.webUrl.indexOf('/', 8));
+
+		const removePage = () => {
+			auth
+				.getAccessToken(resource, auth.service.refreshToken as string, cmd, this.debug)
+				.then((accessToken: string): request.RequestPromise => {
+					if (this.debug) {
+						cmd.log(`Retrieved access token ${accessToken}. Retrieving request digest...`);
+					}
+
+					siteAccessToken = accessToken;
+
+					if (this.verbose) {
+						cmd.log(`Retrieving request digest...`);
+					}
+
+					return this.getRequestDigestForSite(args.options.webUrl, siteAccessToken, cmd, this.debug);
+				})
+				.then((res: ContextInfo): request.RequestPromise => {
+					if (this.debug) {
+						cmd.log('Response:');
+						cmd.log(res);
+						cmd.log('');
+					}
+
+					requestDigest = res.FormDigestValue;
+
+					if (!pageName.endsWith('.aspx')) {
+						pageName += '.aspx';
+          }
+          
+          if (this.verbose) {
+						cmd.log(`Removing page ${pageName}...`);
+					}
+
+					const requestOptions: any = {
+						url: `${args.options
+							.webUrl}/_api/web/getfilebyserverrelativeurl('${serverRelativeSiteUrl}/sitepages/${pageName}')`,
+						headers: Utils.getRequestHeaders({
+							authorization: `Bearer ${siteAccessToken}`,
+							'X-RequestDigest': requestDigest,
+							'X-HTTP-Method': 'DELETE',
+							'content-type': 'application/json;odata=nometadata',
+							accept: 'application/json;odata=nometadata'
+						}),
+						json: true
+					};
+
+					if (this.debug) {
+						cmd.log('Executing web request...');
+						cmd.log(requestOptions);
+						cmd.log('');
+					}
+
+					return request.post(requestOptions);
+				})
+				.then((): void => {
+						if (this.verbose) {
+							cmd.log(vorpal.chalk.green('DONE'));
+						}
+						cb();
+          },
+					(err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb)
+				);
+		};
+
+		if (args.options.confirm) {
+			removePage();
+		} else {
+			cmd.prompt(
+				{
+					type: 'confirm',
+					name: 'continue',
+					default: false,
+					message: `Are you sure you want to remove the page '${args.options.name}'?`
+				},
+				(result: { continue: boolean }): void => {
+					if (!result.continue) {
+						cb();
+					} else {
+						removePage();
+					}
+				}
+			);
+		}
+	}
+
+	public options(): CommandOption[] {
+		const options: CommandOption[] = [
+			{
+				option: '-n, --name <name>',
+				description: 'Name of the page to create'
+			},
+			{
+				option: '-u, --webUrl <webUrl>',
+				description: 'URL of the site where the page should be created'
```

Comment:
Should be `URL of the site where the page should be removed`

Reply:
Done